Code/MinimalLib/jswrapper.cpp

```diff
@@ -553,29 +552,44 @@ EMSCRIPTEN_BINDINGS(RDKit_minimal) {
       .function("straighten_depiction",
                 select_overload<void()>(&JSMol::straighten_depiction))
       .function("straighten_depiction",
-                select_overload<void(bool)>(&JSMol::straighten_depiction));
+                select_overload<void(bool)>(&JSMol::straighten_depiction))
+      .function("get_num_atoms", select_overload<unsigned int(bool) const>(
+                                     &JSMol::get_num_atoms))
+      .function("get_num_atoms",
+                select_overload<unsigned int() const>(&JSMol::get_num_atoms))
+      .function("get_num_bonds", &JSMol::get_num_bonds)
+      .function("get_copy", &JSMol::get_copy, allow_raw_pointers());
 
   class_<JSMolIterator>("MolIterator")
+      .constructor<>()
+      .function("append", &JSMolIterator::append)
+      .function("insert", &JSMolIterator::insert)
+      .function("at", &JSMolIterator::at, allow_raw_pointers())
+      .function("pop", &JSMolIterator::pop, allow_raw_pointers())
```

Comment:
not related to PR

Reply:
Actually this is related, because those methods can be useful when constructing/dealing with a `JSMolterator` to prepare input for MCS.